enhancements/olm/operator-bundle.md

```diff
@@ -88,6 +93,11 @@ The labels will also be put inside a YAML file, as shown below.
 annotations:
   operators.operatorframework.io.bundle.resources: "manifests+metadata"
```

Comment:
It seems like we don't need this field if we have the other two?

Reply:
yep, good point.